neo/core/event.py

```diff
@@ -62,6 +64,10 @@ class Event(BaseNeo, pq.Quantity):
         :description: (str) Text description.
         :file_origin: (str) Filesystem path or URL of the original data file.
 
+    *Optional attributes/properties*:
+        :array_annotations: (dict) Dict mapping strings to numpy arrays containing annotations \
```

Comment:
no '\' here

Reply:
The '\' is masking the newline for building the documentation. If it is removed, this sentence will not be formatted correctly.

Reply:
The '\' is masking the newline for building the documentation. If it is removed, this sentence will not be formatted correctly, so I think it's better to keep it here.